### ISSUE TITLE
Add original author's username to remix data

### ISSUE THREAD

**Issue by gvn** [mozilla/api.webmaker.org#87]
When I do a POST to  `/users/USER/projects/PROJECT/remixes` I need the original author's username to come back in the JSON so I can do attribution.

Blocks: https://github.com/mozilla/webmaker-android/issues/1569

**Comment by gvn** [comment]
Ok, for now I'm going to do another call out to `users/X/projects/X` to get the username.

Adding to the remix response is a nice-to-have.
